cmd/skopeo/utils.go

```diff
@@ -409,3 +411,18 @@ func promptForPassphrase(privateKeyFile string, stdin, stdout *os.File) (string,
 	fmt.Fprintf(stdout, "\n")
 	return string(passphrase), nil
 }
+
+// Filter out refs which are not follow Semantic versioning
+func filterOutRefsNonSemver(imgRefs []types.ImageReference) ([]types.ImageReference, []types.ImageReference) {
```

Comment:
(I’d keep this in `sync.go`; it seems unlikely to be useful for other subcommands.)